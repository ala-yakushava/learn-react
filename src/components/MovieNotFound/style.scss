.MovieNotFound {
  display: flex;
  justify-content: center;
  align-items: center;
  height: 500px;
  padding-top: 200px;
}
