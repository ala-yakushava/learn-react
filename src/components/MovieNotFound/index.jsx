import React from 'react';
import PropTypes from 'prop-types';

import './style.scss';
import Heading from '../Heading';

const MovieNotFound = () => (
  <section className="MovieNotFound" data-testid="movie-not-found">
    <Heading>No Movie Found</Heading>
  </section>
);

Heading.propTypes = {
  onClick: PropTypes.func,
};

Heading.defaultProps = {
  onClick: () => {},
};

export default MovieNotFound;
