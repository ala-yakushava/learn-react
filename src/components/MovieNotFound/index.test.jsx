import React from 'react';
import renderer from 'react-test-renderer';

import MovieNotFound from './index';

describe('Components - MovieNotFound', () => {
  test('should match snapshot', () => {
    const tree = renderer
      .create(<MovieNotFound />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
