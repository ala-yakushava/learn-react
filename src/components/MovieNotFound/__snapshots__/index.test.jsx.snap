// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components - MovieNotFound should match snapshot 1`] = `
<section
  className="MovieNotFound"
  data-testid="movie-not-found"
>
  <div
    className="Heading"
  >
    No Movie Found
  </div>
</section>
`;
